airflow/providers/google/cloud/hooks/datastore.py

```diff
@@ -104,14 +104,16 @@ def begin_transaction(self, project_id: str) -> str:
 
         :param project_id: Google Cloud Platform project ID against which to make the request.
         :type project_id: str
+        :param transaction_options: Options for a new transaction.
+        :type transaction_options: Dict[str, Any]
         :return: a transaction handle.
         :rtype: str
         """
         conn = self.get_conn()  # type: Any
```

Comment:
Is it necessary to add such type notation like `type: Any`? Just asking. :)

Reply:
`Dict[str, Any]` is more informative than `Dict` because you know the type of keys

Reply:
I was thinking about it `conn = self.get_conn()  # type: Any` :)

Reply:
Oh, I think it adds no value, but maybe it was required for mypy to be happy